Codeinwp/raft
# New patterns

### Summary
- Added most of the new patterns as php templates
- Adjusted/added string variables
- Adjusted / added pattern categories

All strings and images in these patterns reference the variables we have created for strings.

### Will affect visual aspect of the product

NO

### Test instructions
- Please review the pattern PHPs
- Install the build and in the pattern inserter see if all the patterns appear and are being imported correctly.


Closes #.
